packages/cozy-harvest-lib/src/services/deleteAccounts.js

```diff
@@ -0,0 +1,13 @@
+import get from 'lodash/get'
+
+import { fetchAccountsForCipherId } from './utils'
+
+const deleteAccounts = async (cozyClient, bitwardenCipherDocument) => {
+  const bitwardenCipherId = get(bitwardenCipherDocument, '_id')
+
+  const accounts = await fetchAccountsForCipherId(cozyClient, bitwardenCipherId)
+
+  await Promise.all(accounts.map(account => cozyClient.destroy(account)))
```

Comment:
As a service I think it would be interesting to add logging ? The minilog instance in src/logger could help ?

Reply:
Yep good idea, thanks. See https://github.com/cozy/cozy-libs/pull/942/commits/c89c753f76477ef0240a28decaf716096b1053e1